fn foo(): Number
   compute_number() # !> Can't find 'compute_number' in this scope
end